rhino/src/main/java/org/mozilla/javascript/typedarrays/NativeTypedArrayView.java

```diff
@@ -290,6 +294,16 @@ private static void defineMethod(
         typedArray.definePrototypeMethod(scope, key, length, target, DONTENUM, DONTENUM | READONLY);
     }
 
+    private static void defineConstructorMethod(
```

Comment:
do we really need to extract this single method into another method that is called only once - to understand what is happening you have to look at two places

Reply:
and it looks like the function declaration is longer than the body itself

Reply:
Fair point. I'm going to use it for `TypedArray.of` too, so that's going to be one more use

Reply:
But I ended up not having this method - you're right, it was pointless.